### ISSUE TITLE
Default root response

### ISSUE THREAD

**Issue by robinvanemden** [MKaptein/streamingbandit#31]
When you currently visit the root of an f/usermanagement branch SB server, you receive 

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/tornado/web.py", line 1488, in _execute
    result = self.prepare()
  File "/usr/local/lib/python3.5/dist-packages/tornado/web.py", line 2223, in prepare
    raise HTTPError(self._status_code)
tornado.web.HTTPError: HTTP 404: Not Found
```

Probably nice to return a general info page, preferably with an overview of the REST interface.

For example:

https://httpbin.org/
https://jsonplaceholder.typicode.com/

**Comment by MKaptein** [comment]
What do you mean by "visit SB"? Go to the address request index.html?

I guess that should have a static page with links to the API docs, the front-end repo, etc.... (I would say not the actual docs themselves)

Correct?

Verstuurd vanaf mijn iPhone

**Comment by robinvanemden** [comment]
Exactly.

**Comment by MKaptein** [comment]
Yes, agreed here; basically all the calls to localhost:8080 that are not part of the REST API should go to a static index.html file (should be simple to arrange in the base handler GET specification). I can write the file...

**Comment by g0ulash** [comment]
Fixed in commit d0f58a2
